server/package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Backend subproject for offline-markdown-previewer",
   "main": "server.js",
   "config": {
-    "port": "8080"
+    "http_port": "8080",
+    "ws_port": "8081"
```

Comment:
if you're using this, need to use that in client api as well, I think we might need to use `DefinePlugin` or `EnvironmentPlugin`